crowdstrikeoauthapi_connector.py

```diff
@@ -1668,6 +1681,74 @@ def _handle_list_detections(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_detections_details(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        ids = [det_id.strip() for det_id in param.get("detection_ids").split(',')]
+        ids = list(set(filter(None, ids)))
```

Comment:
```suggestion
        ids = list(set(det_id.strip() for det_id in param.get("detection_ids").split(',') if det_id.strip()))
```

Reply:
Also - this is done all over this file, I think this should be a function/method

Reply:
Done